nav2_system_tests/src/costmap_filters/speed_local_params.yaml

```diff
@@ -0,0 +1,318 @@
+amcl:
```

Comment:
Can you highlight why both local and global testing? We chatted about wanting to have this in only 1 of the 2 costmaps (because if both, they might be overwriting each other), so maybe it would be good if we set a precedent about which we use by convention.

Reply:
Though I agree for testing it would be good to do local either way (to test rolling costmap and different frame)

Reply:
Yes, this is made to test speed filter for different cases: one for local another for global costmap usage. They are not intended to be used at the same time (as made for keepout zones filter).

Reply:
I was just trying to understand why we have 2 param files and what each test was specifically covering.

Reply:
They are covering two types of usage of `SpeedFilter`:
- `test_speed_filter_global` is made to check that speed filter will work with global costmaps (costmap, map and mask are in the same frame).
- `test_speed_filter_local` is made to check speed filter functionality with local costmaps. There are map and mask published in `map` frame while costmap is in `odom`. This test covers the TF-related logic (similar logic as in #2053, but for SpeedFilter).

As we discussed before, unlike `KeepoutFilter` two cases (local+global) can not be run simultaneously at the same time for `SpeedFilter`